docker/docker-compose.dev.yml

```diff
@@ -79,9 +79,14 @@ services:
   immich-machine-learning:
     container_name: immich_machine_learning
     image: immich-machine-learning-dev:latest
+    extends:
```

Comment:
only the dev file, yet this should be commented out by default as some dev's won't have a cuda device

Reply:
Oops, this was probably committed by accident while I was testing haha